### PR TITLE
[RDBMS] `az postgres flexible-server long-term-retention start/pre-check/list/show`: New commands to support starting, prechecking, listing, showing long-term-retention backups on PostgreSql Flex Server

### PR DESCRIPTION
**Pull request by mattboentoro**
[RDBMS] BREAKING CHANGE: `az postgres flexible-server long-term-retention start/pre-check/list/show`: New commands to support starting, prechecking, listing, showing long-term-retention backups on PostgreSql Flex Server

**Related command**

**Description**

New commands to support starting, prechecking, listing, showing long-term-retention backups on PostgreSql Flex Server

**Testing Guide**

Unit test, visual test

**History Notes**

[RDBMS] `az postgres flexible-server long-term-retention start/pre-check/list/show`: New commands to support long-term-retention backups on PostgreSql Flex Server

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [X] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [X] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/azure-cli/azure/cli/command_modules/rdbms/_helptext_pg.py [hunk 1]
```diff
@@ -768,15 +768,15 @@
...
   - name: Precheck command.
-    text: az postgres flexible-server long-term-retention pre-check -g resource-group-name -b backup-name -n server-name 
+    text: az postgres flexible-server long-term-retention pre-check -g resource-group-name -b backup-name -n server-name
 """
 
 helps['postgres flexible-server long-term-retention start'] = """
 type: command
 short-summary: Start long-term-retention backup for a flexible server.
 examples:
   - name: Create a backup.
```
Comment (nasc17) [inline src/azure-cli/azure/cli/command_modules/rdbms/_helptext_pg.py:778]: Should this be different

File: src/azure-cli/azure/cli/command_modules/rdbms/_helptext_pg.py [hunk 2]
```diff
@@ -758,6 +758,43 @@
...
 """
 
+helps['postgres flexible-server long-term-retention'] = """
+type: group
+short-summary: Manage flexible server long-term-retention backups.
+"""
+
+helps['postgres flexible-server long-term-retention pre-check'] = """
+type: command
+short-summary: performs all the checks that are needed for the subsequent long-term-retention backup operation to succeed.
```
Comment (nasc17) [inline src/azure-cli/azure/cli/command_modules/rdbms/_helptext_pg.py:768]: Uppercase sentence start

File: src/azure-cli/azure/cli/command_modules/rdbms/_helptext_pg.py (hunk 1 above)
Comment (nasc17) [reply src/azure-cli/azure/cli/command_modules/rdbms/_helptext_pg.py:778]: Also maybe in Summary explain what is sas url

**Comment by evelyn-ys** [comment]
@mattboentoro Pls resolve conflict

**Comment by nasc17** [comment]
@evelyn-ys Please review, thank you

**Review by evelyn-ys — APPROVED** [review]

**Pull request merged**